.github/workflows/test.yml

```
@@ -78,7 +78,7 @@ jobs:
       - name: Assemble
         run: ./gradlew assemble --parallel
       - name: Build scheduling procedure jars for testing
-        run: ./gradlew procedural:examples:foo-procedures:buildAllSchedulingProcedureJars
+        run: ./gradlew e2e-test:build && ./gradlew e2e-tests:buildAllSchedulingProcedureJars
```

I believe we're already running the annotation processing on these example procedures in the `./gradlew assemble --parallel` step above

```suggestion
        run: ./gradlew e2e-tests:buildAllSchedulingProcedureJars
```

---

done